Library/Variable.php

```diff
@@ -1025,12 +1025,7 @@ public function isComplex()
      */
     public function isSuperGlobal()
     {
-        if (!$this->isExternal) {
-            return false;
-        }
-
-        $name = $this->getName();
-        return isset(self::$superglobals[$name]);
+        return isset($this->superglobals[$this->name]);
```

Comment:
@sjinks We should'n use varables like `_POST` or `_SESSION` as internal variables. They always  should be external.